changelog/_unreleased/2025-07-31-add-mediaEntity-to-resolveremotethumbnailurlextension.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Add MediaEntity to ResolveRemoteThumbnailUrlExtension
+author: Sascha Heilmeier
+author_email: sascha.heilmeier@netlogix.de
+author_github: @scarbous
+---
+# Core
+* Added `Shopware\Core\Content\Media\MediaEntity` to `Shopware\Core\Content\Media\Extension\ResolveRemoteThumbnailUrlExtension` to allow individual media handling.
+* Added `mimeType` to `Shopware\Core\Content\Media\Core\Params\UrlParams` 
+* Changed `Shopware\Core\Content\Media\Extension\ResolveRemoteThumbnailUrlExtension` to allow to skip thumbnail loading by returning `null` as result
```

Comment:
sorry, I was mainly concentrating on the code :see_no_evil: but as we introduced some deprecations/breaks for the next major version, we need to extend the changelog. 

add a section `# Upgrade Information` where you describe what was now deprecated and what the alternative is

after that the section `# Next Major Version Changes` which will be part of the 6.8 upgrade guide and which tells, what was removed and what to use instead

Reply:
Thank you for your support 🙏🏼

I hope I get it right 😊